cmd/main.go

```diff
@@ -13,7 +13,7 @@ func main() {
 	r := router.Router()
 	port := os.Getenv("PORT")
 	if port == "" {
-		port = "9090"
+		port = "9091"
```

Comment:
Please revert this.

Reply:
I have changed this because usermgmt service runs on 9090

Reply:
How does execution of user management service impact execution of this service? Why not change the user management service to run on 9091?